src/vma/proto/dst_entry_udp_mc.cpp

```diff
@@ -81,6 +81,24 @@ bool dst_entry_udp_mc::conf_l2_hdr_and_snd_wqe_ib()
 	return ret_val;
 }
 
+void dst_entry_udp_mc::set_src_addr()
+{
+	m_pkt_src_ip = INADDR_ANY;
+	
+	if (m_bound_ip) {
+		m_pkt_src_ip = m_bound_ip;
```

Comment:
If that case is allowed, then set_bound_addr will call set_state(false), as result resolve_net_dev will called again and assign m_route_src_ip one more time (currently we are not supporting multiple bind since kernel doesn't allow it).